docs/api/regressor/alphas.rst

```diff
@@ -9,16 +9,18 @@ The AlphaSelection Visualizer demonstrates how different values of alpha influen
 
 .. code:: python
 
-    # Load the data
+    # Load the regression data set
```

Comment:
In a majority of our code samples I noticed that we indicate whether we are loading a dataset best-suited for regression or classification models, so I added that information here.